_includes/head.html

```diff
@@ -15,6 +15,7 @@
 
   <link rel="stylesheet" href="{{ "/css/main.css" | prepend: site.baseurl }}">
   <link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
Let's move this line before we call our `main.css` file. 
In this way our css will be called after the library one. 
This means our styles will be always the last one so will be the one that actually is applied. (so if we want to override some libraries style, we can do it ;) )

Reply:
Got it!